src/components/VendorDetail/VendorDetail.tsx

```diff
@@ -121,9 +141,9 @@ const VendorDetail: React.FC<VendorDetailProps> = ({ vendor, user, sendStats })
                             </Tag>
                             : <Tag
                                 variant='solid'
-                                background='#E13784'
+                                background={'#e13784'}
```

Comment:
zkusil bych použít nějákou pink.xxx a v hoveru nějákou tmavší pink.xxx at se nemusí improtovat nějáký package na to